src/MCPClient/lib/clientScripts/copyTransferSubmissionDocumentation.py

```diff
@@ -41,12 +42,32 @@
     submissionDocumentationDirectory = sys.argv[2]
     sharedPath = sys.argv[3]
 
-    transfer_locations = File.objects.filter(removedtime__isnull=True, sip_id=sipUUID, transfer__currentlocation__isnull=False).values_list('transfer__currentlocation').distinct()
+    transfer_locations = File.objects.filter(removedtime__isnull=True, sip_id=sipUUID, transfer__currentlocation__isnull=False).values_list('transfer__currentlocation', flat=True).distinct()
 
-    for transferLocation, in transfer_locations:
+    sip = SIP.objects.get(uuid=sipUUID)
+    aip_mets_name = 'METS.' + sipUUID + '.xml'
+
+    for transferLocation in transfer_locations:
         transferLocation = transferLocation.replace("%sharedPath%", sharedPath)
         transferNameUUID = os.path.basename(os.path.abspath(transferLocation))
-        src = os.path.join(transferLocation, "metadata/submissionDocumentation")
+        src = os.path.join(transferLocation, "metadata", "submissionDocumentation")
         dst = os.path.join(submissionDocumentationDirectory, "transfer-%s" % (transferNameUUID))
-        print(src, " -> ", dst, file=sys.stderr)
-        shutil.copytree(src, dst)
+
+        # For reingest, ignore this transfer's submission docs, only copy submission docs from the original Transfer
+        if 'REIN' in sip.sip_type:
```

Comment:
It'd be nice to make use of enums for safety, e.g. https://github.com/5monkeys/django-enumfield